File: Java-Oracle/c2f/C2F.java
import java.lang.System;

public class C2F {
    public static void main(String[] args) {
        System.out.println(ConvertCelciusToFahrenheit(0));
    }

    public static double ConvertCelciusToFahrenheit(double celcius) {
        return celcius * 9/5 + 32;
    }
}

File: CSharp-SQLite-APP/SQLiteFunctions/SQLiteFunctions/SQLiteFunctions/Functions/C2F.cs
using System;
using System.Data.SQLite;

namespace SQLiteFunctions.Functions
{
    [SQLiteFunction(Name = "c2f", Arguments = 1, FuncType = FunctionType.Scalar)]
    public class C2F : SQLiteFunction
    {
        public string ConstructCommand()
        {
            Console.Write("Celcius degree: ");
            var input = Console.ReadLine();

            return $"c2f({input})";
        }

        private double GetFahrenheit(double celcius)
        {
            return celcius * 9 / 5 + 32;
        }

        public override object Invoke(object[] args)
        {
            return GetFahrenheit(double.Parse(args[0].ToString()));
        }
    }
}
